code/modules/overmap/exoplanets/decor/turfs/water.dm

```diff
@@ -1,12 +1,98 @@
 /turf/simulated/floor/exoplanet/water
 	does_footprint = FALSE
+	footstep_sound = /singleton/sound_category/water_footstep
+	var/deep = TRUE //too deep to keep your head above it?
+	var/obj/effect/water_effect/water_overlay
+	movement_cost = 4
+	var/numobjects = 0
```

Comment:
DMDoc these variables

generally keep new variables at the bottom and already-defined one at the top too